cmake/CxxThirdParty.cmake

```
@@ -50,6 +50,45 @@ cpmaddpackage(
   https://github.com/nlohmann/json/releases/download/v3.11.3/include.zip
   SYSTEM)
 
+if(${NEOFOAM_WITH_ADIOS2})
+
+  set(ADIOS2_KOKKOS_PATCH git apply ${CMAKE_CURRENT_SOURCE_DIR}/cmake/patches/adios2_kokkos.patch)
+
+  set(ADIOS2_OPTIONS
+      "BUILD_TYPE Release"
```

Maybe it is better if we use the  `CMAKE_BUILD_TYPE` here?

```suggestion
      "BUILD_TYPE ${CMAKE_BUILD_TYPE}"
```